commitrmnocb/observation_test.go

```diff
@@ -223,10 +225,12 @@ func Test_ObserveOffRampNextSeqNums(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			ctx := context.Background()
 			var nodeID commontypes.OracleID = 1
-			reader := mocks.NewCCIPReader()
-			reader.On(
-				"NextSeqNum", ctx, tc.knownSourceChains,
-			).Return(tc.nextSeqNums, tc.nextSeqNumsError)
+			reader := reader_mock.NewMockCCIP(t)
+			if tc.supportsDestChain && tc.supportsDestChainError == nil && tc.knownSourceChainsError == nil {
```

Comment:
@rstout I switched our manually written CCIP reader mock with a generated version which automatically calls `AssertExpectations`. Several of these tests needed to be updated to account for the stricter assertions.